cdc/redo/writer/file.go

```diff
@@ -38,17 +38,15 @@ import (
 	"github.com/pingcap/tiflow/cdc/model"
 	"github.com/pingcap/tiflow/cdc/redo/common"
 	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/fsutil"
 	"github.com/pingcap/tiflow/pkg/uuid"
 )
 
 const (
 	// pageBytes is the alignment for flushing records to the backing Writer.
 	// It should be a multiple of the minimum sector size so that log can safely
 	// distinguish between torn writes and ordinary data corruption.
-	pageBytes = 8 * common.MinSectorSize
-)
-
-const (
+	pageBytes                = 8 * common.MinSectorSize
 	defaultFlushIntervalInMs = 1000
```

Comment:
2000？

Reply:
`defaultFlushIntervalInMs` here is never used in the code base. So I removed it.